package.json

```diff
@@ -30,7 +30,7 @@
   },
   "scripts": {
     "dev:chrome": "web-ext run --source-dir=build/chrome --target=chromium --start-url=examples/sample-dapp/index.html",
-    "dev:firefox": "web-ext run --source-dir=build/firefox --devtools --target=firefox-desktop --start-url=examples/sample-dapp/index.html",
+    "dev:firefox": "web-ext run --source-dir=build/firefox --firefox=firefoxdeveloperedition --start-url=examples/sample-dapp/index.html",
```

Comment:
@dalebennett1992 I inadvertently committed this, maybe that breaks firefox tests